conf/deployment/vsphere/ai_1az_rhcos_vsan_3m_3w_hci.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# This config is suppose to work on DC-CP, DC-ECO, DC-PS
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'vsphere'
+  deployment_type: 'ai'
+  worker_replicas: 3
+  master_replicas: 3
+  master_num_cpus: '16'
+  worker_num_cpus: '16'
+  master_memory: '65536'
+  compute_memory: '65536'
```

Comment:
is it required for both master and worker has same CPU's and memory. Usally master will have less resource requirement than workers.

Reply:
This is the requirement for Provider/Client deployment (for the normal deployment there is the first configuration file `conf/deployment/vsphere/ai_1az_rhcos_vsan_3m_3w.yaml` with less CPU's and memory for master nodes).